src/main/java/com/tabnine/TabNineCompletionContributor.java

```diff
@@ -233,8 +237,9 @@ public void fillCompletionVariants(CompletionParameters parameters, CompletionRe
         }
     }
 
-    @Override
-    public boolean invokeAutoPopup(@NotNull PsiElement position, char typeChar) {
-        return true;
-    }
+//    @Override
```

Comment:
@avnerbarr I think we need it, but not after every char. We should probably include a-z, digits, <space>, =

Thoughts?

Reply:
I played with it and the default behaviour was fine on my system (2020.2.1 CE). 

The method is deprecated and they suggest not using it.

```/**
   * @deprecated Don't use this method, because {@code position} can come from uncommitted PSI and be totally unrelated to the code being currently in the document/editor.
   * Please consider using {@link com.intellij.codeInsight.editorActions.TypedHandlerDelegate#checkAutoPopup} instead.
   */
  @Deprecated
  public boolean invokeAutoPopup(@NotNull PsiElement position, char typeChar) {
    return false;
  }
```

Reply:
OK, cool. Playing with it :+1:

Reply:
Are we sure about this?